homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index_cards.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Cards</title>
+        <link rel="stylesheet" href="styles.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;800&family=Roboto:wght@500&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;800&family=Roboto:wght@400&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <header class="header">
+            <div class="header__logo">
+                <svg class="logo">
+                    <use xlink:href="sprite.svg#logo"></use>
+                </svg>
+            </div>
+            <div class="header__navigation">
+                <ul class="navigation">
+                    <li class="navigation__line">
+                        Games
+                    </li>
+                    <li class="navigation__line">
+                        <a class="navigation__link" href="index_about.html">About</a>
+                    </li>
+                </ul>
+            </div>
+        </header>
+        <main class="main games__main">
+            <h1>Study project "Games"</h1>
+            <article class="card-container">
+                <section class="card">
+                    <div class="card__image">
+                        <img src="images/Rectangle 7.png" alt="image">
```

Comment:
Add more correct names for your images. Lower case, no spaces and a name with meaning is better.